### PR TITLE
[Blog] What is a Slice

### PR DESCRIPTION
**Pull request by verma-kunal**
Signed-off-by: Kunal Verma <vkunal321@gmail.com>

This is the 1st draft of the blog - What is a Slice? (Overlay Network & How it works in KubeSlice) which is to be published next week. 

P.S - I have written a technical blog after a long time so, would definitely need suggestions/feedbacks from the community for more improvement.
Thanks

### CODE REVIEW

**Comment by verma-kunal** [comment]
cc @divya-mohan0209 @pdorbala would love to hear your suggestions and feedbacks here! Would improve the article accordingly
Thanks :)

File: blog/what-is-a-Slice.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Each network slice is logically separated from each other, is an independent part of the network and targets different services with different requirements at the same time. This enables to build and manage a network, which has the capabilities to meet and exceed the emerging requirements from a wide range of users and that too, at the same time - The concept of multi-tenancy.
+
+### Concept of a Slice in KubeSlice
+
+In terms of Kubernetes, a Slice is a [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) which can be defined and it basically describes a way to implement multi-tenancy in a cluster. Multi-tenancy is a concept wherein a single instance of an application or a single Kubernetes cluster can serve multiple user (called tenants) at the same time and is able to withstand that much amount of load.
```
Comment (neelanjan00) [inline blog/what-is-a-Slice.mdx:27]:
```suggestion
In terms of Kubernetes, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) which can be defined, and it basically describes a way to implement multi-tenancy in a cluster. Multi-tenancy is a concept wherein a single instance of an application or a single Kubernetes cluster can serve multiple users (called tenants) at the same time and is able to withstand that much amount of load.
```

File: blog/what-is-a-Slice.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+You might be wondering, how does the creation of a Slice an essential part of KubeSlice and helps in achieving the goals of multi-tenancy?
+
+Well, KubeSlice actually creates these logical application **slice boundaries** in a cluster, that allows pods and services to communicate efficiently within a cluster, across clusters as well as on cloud or any other data centres as well. In order to communicate in a multi-cluster architecture, an overlay network (an application slice) is created which provides seamless connectivity between the pods of an application running in multiple clusters.
```
Comment (neelanjan00) [inline blog/what-is-a-Slice.mdx:31]:
```suggestion
Well, KubeSlice actually creates these logical application **slice boundaries** in a cluster, that allows pods and services to communicate efficiently within a cluster, across clusters as well as on the cloud or any other data center as well. In order to communicate in a multi-cluster architecture, an overlay network (an application slice) is created which provides seamless connectivity between the pods of an application running in multiple clusters.
```

File: blog/what-is-a-Slice.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Therefore, due to this **slice overlay network**, pods are able to communicate seamlessly across cluster boundaries.
+
+You may ask a question here, what are some of the benefits we get due to the creation of the slice overlay network?
```
Comment (neelanjan00) [inline blog/what-is-a-Slice.mdx:35]:
```suggestion
You may ask a question here, what are some of the benefits that we get due to the creation of the slice overlay network?
```

File: blog/what-is-a-Slice.mdx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+There are a lot of advantages achieved here:
+
+- **Better Resiliency** - In layman language, Resiliency is the ability an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services get better resiliency
```
Comment (neelanjan00) [inline blog/what-is-a-Slice.mdx:39]:
```suggestion
- **Better Resiliency** - In layman's language, Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services get better resiliency
```

File: blog/what-is-a-Slice.mdx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- **Traffic Control** - The ability to control traffic across nodes and clusters. The traffic control policy is maintained for all application pods on that slice.
```
Comment (neelanjan00) [inline blog/what-is-a-Slice.mdx:40]: You may also mention how it handles the challenges of overlapping IPs.

Comment (verma-kunal) [reply blog/what-is-a-Slice.mdx:40]: Sure @neelanjan00 ! Would add this over the weekend & thanks for the above changes :)

File: blog/what-is-a-Slice.mdx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+title: What is a Slice?
+description: What is a Slice? (Overlay Network & How it works in KubeSlice)
+slug: what-is-a-slice
+authors:
+  - name: Kunal Verma
+    url: https://github.com/verma-kunal
+hide_table_of_contents: false
+---
+
+[KubeSlice](https://kubeslice.io/documentation/open-source/0.3.0) is fairly a new project/tool in the cloud-native ecosystem which addresses the area and issues of [multi-tenancy](https://digitalguardian.com/blog/saas-single-tenant-vs-multi-tenant-whats-difference) and [multi-cluster](https://www.mirantis.com/cloud-native-concepts/getting-started-with-kubernetes/what-is-kubernetes-multi-cluster/) environment in Kubernetes. By definition, KubeSlice combines network, application, and deployment services in a framework to create multi-tenancy in a Kubernetes cluster and extends it to multi-cluster architecture.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:11]:
```suggestion
[KubeSlice](https://kubeslice.io/documentation/open-source/0.3.0) is a new project in the cloud-native ecosystem that addresses the issues of a multi-cluster multi-tenant environment in Kubernetes.
By definition, KubeSlice combines network, application, and deployment services in a framework to create multi-tenancy in a Kubernetes cluster and extends it to a multi-cluster architecture.
```

Comment (divya-mohan0209) [reply blog/what-is-a-Slice.mdx:11]: We'd like to limit linking within blogposts/articles to the official Kubernetes documentation OR our documentation. Third party content should be as limited as possible.

File: blog/what-is-a-Slice.mdx [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+You might be wondering, how does it do this? Basically, KubeSlice achieves this by creating logical application **Slice** boundaries that allow pods and services to communicate efficiently across clusters, clouds, and data centers.  
+
+If you see here, the term **application slice** or **Slice** is mentioned. This is one of the key components in KubeSlice and in this particular article we will understand what exactly is a Slice and how it enables KubeSlice to tackle the problems that Kubernetes have in achieving multi-tenancy on a multi-cluster architecture.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:15]:
```suggestion
A **slice** is one of the key components and in this particular article we will understand what exactly it is and how it enables KubeSlice to tackle the problems of achieving multi-tenancy on a multi-cluster architecture in Kubernetes.
```

File: blog/what-is-a-Slice.mdx [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+slug: what-is-a-slice
+authors:
+  - name: Kunal Verma
+    url: https://github.com/verma-kunal
+hide_table_of_contents: false
+---
+
+[KubeSlice](https://kubeslice.io/documentation/open-source/0.3.0) is fairly a new project/tool in the cloud-native ecosystem which addresses the area and issues of [multi-tenancy](https://digitalguardian.com/blog/saas-single-tenant-vs-multi-tenant-whats-difference) and [multi-cluster](https://www.mirantis.com/cloud-native-concepts/getting-started-with-kubernetes/what-is-kubernetes-multi-cluster/) environment in Kubernetes. By definition, KubeSlice combines network, application, and deployment services in a framework to create multi-tenancy in a Kubernetes cluster and extends it to multi-cluster architecture.
+
+You might be wondering, how does it do this? Basically, KubeSlice achieves this by creating logical application **Slice** boundaries that allow pods and services to communicate efficiently across clusters, clouds, and data centers.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:13]:
```suggestion
KubeSlice achieves this by creating logical application **Slice** boundaries that allow pods and services to communicate efficiently across clusters, clouds, and data centers.  
```

File: blog/what-is-a-Slice.mdx [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+You might be wondering, how does it do this? Basically, KubeSlice achieves this by creating logical application **Slice** boundaries that allow pods and services to communicate efficiently across clusters, clouds, and data centers.  
+
+If you see here, the term **application slice** or **Slice** is mentioned. This is one of the key components in KubeSlice and in this particular article we will understand what exactly is a Slice and how it enables KubeSlice to tackle the problems that Kubernetes have in achieving multi-tenancy on a multi-cluster architecture.
+
+So let’s start!
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:17]:
```suggestion
```

File: blog/what-is-a-Slice.mdx [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+### What is a Slice?
+
+One of the best real-world examples out there of a Slice is the concept of **Network Slicing** in 5G networks today. Network Slicing is a method of creating multiple unique **logical** and **virtualized** networks over a common (shared) infrastructure. You all might be familiar with remote Virtual Machines, right? Well, these are virtual servers running on the cloud or containers and are not physically present. Similarly, a network slice is carved out of a large and common infrastructure.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:21]:
```suggestion
One of the best real-world examples out there of a Slice is the concept of **Network Slicing** in 5G networks today. 
Network Slicing is a method of creating multiple unique **logical** and **virtualized** networks over a common (shared) infrastructure. 
Similar to virtual machines, a network slice is carved out of a large and common infrastructure. 
```

File: blog/what-is-a-Slice.mdx [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+So let’s start!
+
+### What is a Slice?
+
+One of the best real-world examples out there of a Slice is the concept of **Network Slicing** in 5G networks today. Network Slicing is a method of creating multiple unique **logical** and **virtualized** networks over a common (shared) infrastructure. You all might be familiar with remote Virtual Machines, right? Well, these are virtual servers running on the cloud or containers and are not physically present. Similarly, a network slice is carved out of a large and common infrastructure. 
+
+Each network slice is logically separated from the other, is an independent part of the network, and targets different services with different requirements at the same time. This enables to build and manage a network, which has the capabilities to meet and exceed the emerging requirements from a wide range of users and that too, at the same time: the concept of multi-tenancy.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:23]:
```suggestion
Each network slice is logically separated from the other, is an independent part of the network, and targets different services
with different requirements at the same time. This enables to build and manage a network that has the capabilities 
to meet the emerging requirements from a wide range of users and enable multi-tenancy in a multi-cluster Kubernetes setup.
```

File: blog/what-is-a-Slice.mdx [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+
+### Concept of a Slice in KubeSlice
+
+In terms of Kubernetes, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) which can be defined, and it basically describes a way to implement multi-tenancy in a cluster. Multi-tenancy is a concept wherein a single instance of an application or a single Kubernetes cluster can serve multiple users (called tenants) at the same time and is able to withstand that much amount of load.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:27]:
```suggestion
In KubeSlice, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) defined to implement multi-tenancy in a cluster. [Multi-tenancy](link to Kubernetes guide on multi-tenancy) is when a single instance of an application or a single Kubernetes cluster can serve multiple users (called tenants) at the same time without the problems of noisy/nosey neighbour, policy and configuration drift, and resource hogging creeping in.
```

File: blog/what-is-a-Slice.mdx [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+
+You might be wondering, how does the creation of a Slice an essential part of KubeSlice and helps in achieving the goals of multi-tenancy?
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:29]:
```suggestion
```

File: blog/what-is-a-Slice.mdx [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+
+Well, KubeSlice actually creates these logical application **slice boundaries** in a cluster, that allows pods and services to communicate efficiently within a cluster, across clusters as well as on the cloud or any other data center as well. In order to communicate in a multi-cluster architecture, an overlay network (an application slice) is created which provides seamless connectivity between the pods of an application running in multiple clusters.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:31]:
```suggestion
KubeSlice creates logical application **slice boundaries** in a cluster that allows pods and services to communicate efficiently within a cluster, across clusters on the cloud or datacenters. In order to communicate within a multi-cluster architecture, an overlay network (an application slice) following the RFC1918 protocol is created by KubeSlice that provides seamless connectivity between the pods of an application running in multiple clusters.
```

File: blog/what-is-a-Slice.mdx [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+
+Therefore, due to this **slice overlay network**, pods are able to communicate seamlessly across cluster boundaries.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:33]:
```suggestion
```

File: blog/what-is-a-Slice.mdx [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+
+You may ask a question here, what are some of the benefits that we get due to the creation of the slice overlay network?
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:35]:
```suggestion
```

File: blog/what-is-a-Slice.mdx [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+
+There are a lot of advantages achieved here:
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:37]:
```suggestion
There are a lot of advantages achieved with this way of implementing multi-cluster multi-tenancy in Kubernetes:
```

File: blog/what-is-a-Slice.mdx [hunk 18]
```diff
@@ -0,0 +1,47 @@
...
+
+- **Better Resiliency** - In layman's language, Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services get better resiliency
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:39]: Could we go a bit further and explain how they become more resilient? In the sense, how does the overlay network actually help with bettering resiliency? Does it provide isolation/failover capabilities? We need to explain that here.
```suggestion
- **Better Resiliency** - Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services are able to achieve resiliency.
```

File: blog/what-is-a-Slice.mdx [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+- **Traffic Control** - The ability to control traffic across nodes and clusters. The traffic control policy is maintained for all application pods on that slice.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:40]: Need to elaborate on this further, as well. How do we achieve control? Link to the documentation.
```suggestion
- **Traffic Control** - The ability to control traffic across nodes and clusters. The traffic control policy is maintained for all application pods on that slice.
```

File: blog/what-is-a-Slice.mdx [hunk 20]
```diff
@@ -0,0 +1,47 @@
...
+- **Better App-to-App Connectivity** - Application pods on a Slice have better connectivity between them across multiple cluster boundaries.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:41]: Again - why do they have better connectivity? You need to explain that within a slice the overlay network provides a dedicated connection & as a result, for establishing connection between applications on a slice there is no requirement for the request to traverse through the internet. That type of connectivity is known as east-west connectivity.
```suggestion
- **Better App-to-App Connectivity** - Application pods on a Slice have better connectivity between them across multiple cluster boundaries.
```

File: blog/what-is-a-Slice.mdx [hunk 21]
```diff
@@ -0,0 +1,47 @@
...
+
+And we are at the end of this article. There is a lot to cover in terms how KubeSlice works and can be beneficial for your product and organisation, but let’s leave that for another day.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:43]:
```suggestion
```

File: blog/what-is-a-Slice.mdx [hunk 22]
```diff
@@ -0,0 +1,47 @@
...
+
+If you want to explore more about KubeSlice, do refer the [official documentation](https://kubeslice.io/documentation/open-source/0.3.0/). If you face any issue while following along or want to know more about the project, we encourage you to raise questions in the *`#kubeslice`* channel on the [Kubernetes slack](https://slack.k8s.io/) where we hang out.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:45]:
```suggestion
If you want to explore more about KubeSlice, do refer to the [official documentation](https://kubeslice.io/documentation/open-source/0.3.0/). If you face any issue while following along or want to know more about the project, we encourage you to raise questions in the *`#kubeslice`* channel on the [Kubernetes slack](https://slack.k8s.io/) where we hang out.
```

File: blog/what-is-a-Slice.mdx [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+
+Until next time!
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:47]:
```suggestion
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Initial set of comments.

File: blog/what-is-a-Slice.mdx (hunk 19 above)
Comment (verma-kunal) [reply blog/what-is-a-Slice.mdx:42]: Sure @divya-mohan0209 . Will add this

File: blog/what-is-a-Slice.mdx (hunk 20 above)
Comment (verma-kunal) [reply blog/what-is-a-Slice.mdx:43]: Noted! Will work on this

**Comment by verma-kunal** [comment]
@divya-mohan0209 made the suggested changes! Lemme know if any more improvement is needed
Thanks

File: blog/what-is-a-Slice.mdx [hunk 24]
```diff
@@ -0,0 +1,45 @@
...
+Network Slicing is a method of creating multiple unique **logical** and **virtualized** networks over a common (shared) infrastructure. 
+Similar to virtual machines, a network slice is carved out of a large and common infrastructure. 
+
+Each network slice is logically separated from the other, is an independent part of the network, and targets different services
+with different requirements at the same time. This enables to build and manage a network that has the capabilities 
+to meet the emerging requirements from a wide range of users and enable multi-tenancy in a multi-cluster Kubernetes setup.
+
+### Concept of a Slice in KubeSlice
+
+In KubeSlice, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) defined to implement multi-tenancy in a cluster. [Multi-tenancy](link to Kubernetes guide on multi-tenancy) is when a single instance of an application or a single Kubernetes cluster can serve multiple users (called tenants) at the same time without the problems of noisy/nosey neighbour, policy and configuration drift, and resource hogging creeping in.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:31]:
```suggestion
In KubeSlice, a Slice is a Kubernetes [custom resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) defined to implement multi-tenancy in a cluster. 
[Multi-tenancy](https://kubernetes.io/docs/concepts/security/multi-tenancy/) is when a single instance of an application or a single Kubernetes cluster 
can serve multiple users (called tenants) at the same time without the problems of noisy/nosey neighbour, 
policy and configuration drift, and resource hogging creeping in.
```

File: blog/what-is-a-Slice.mdx [hunk 25]
```diff
@@ -0,0 +1,45 @@
...
+
+
+KubeSlice creates logical application **slice boundaries** in a cluster that allows pods and services to communicate efficiently within a cluster, across clusters on the cloud or datacenters. In order to communicate within a multi-cluster architecture, an overlay network (an application slice) following the RFC1918 protocol is created by KubeSlice that provides seamless connectivity between the pods of an application running in multiple clusters.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:34]:
```suggestion
KubeSlice creates logical application **slice boundaries** in a cluster that allows pods and services to 
communicate efficiently within a cluster, across clusters on the cloud or datacenters. In order to communicate 
within a multi-cluster architecture, an overlay network (an application slice) following the RFC1918 protocol is 
created by KubeSlice that provides seamless connectivity between the pods of an application running in multiple 
clusters. Existing intra-cluster communication remains local to the cluster utilizing each pod’s CNI interface. 
Kubeslice provides isolation of network traffic between clusters by creating an overlay network for inter-cluster communication.
```

File: blog/what-is-a-Slice.mdx [hunk 26]
```diff
@@ -0,0 +1,45 @@
...
+
+
+
+There are a lot of advantages achieved with this way of implementing multi-cluster multi-tenancy in Kubernetes:
+
+- **Better Resiliency** - Resiliency is the ability of an application to react to problems in one of its components and still provide the best possible service. By deploying an application across clusters and connecting using the slice overlay network, application services are able to achieve resiliency.
+- **Traffic Control** - The ability to control traffic across nodes and clusters. The traffic control policy is maintained for all application pods on that slice. To know more about how do we achieve traffic control, refer to the [official documentation](https://kubeslice.io/documentation/open-source/0.3.0/).
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:41]: Could we specifically point to the page mentioning traffic control here?

Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:41]:
```suggestion
- **Traffic Control** - The traffic control policy is maintained for all application pods on a slice. To know more about how we achieve traffic control, refer to the [official documentation](https://kubeslice.io/documentation/open-source/0.3.0/).
```

File: blog/what-is-a-Slice.mdx [hunk 27]
```diff
@@ -0,0 +1,45 @@
...
+- **Better App-to-App Connectivity** - Application pods on a Slice have better connectivity between them across multiple cluster boundaries. This is due to the creation of an overlay network for inter-cluster communication. This overlay network provides a dedicated connection within a slice, thus there is no need for the request to transit the internet to establish a connection between apps on a slice. This type of connectivity is referred to as East-West connectivity.
+
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:43]: We haven't spoken about network isolation/overlapping IPs.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Second set of feedback.

File: blog/what-is-a-Slice.mdx [hunk 28]
```diff
@@ -0,0 +1,53 @@
...
+description: What is a Slice? (Overlay Network & How it works in KubeSlice)
+slug: what-is-a-slice
+authors:
+  - name: Kunal Verma
+    url: https://github.com/verma-kunal
+hide_table_of_contents: false
+---
+
+[KubeSlice](https://kubeslice.io/documentation/open-source/0.3.0) is a new project in the cloud-native ecosystem that addresses the issues of a multi-cluster multi-tenant environment in Kubernetes.
+By definition, KubeSlice combines network, application, and deployment services in a framework to create multi-tenancy in a Kubernetes cluster and extends it to a multi-cluster architecture.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:12]:
```suggestion
The project combines network, application, and deployment services in a framework to create multi-tenancy in a Kubernetes cluster and extends it to a multi-cluster architecture.
```

File: blog/what-is-a-Slice.mdx [hunk 29]
```diff
@@ -0,0 +1,53 @@
...
+
+KubeSlice achieves this by creating logical application **Slice** boundaries that allow pods and services to communicate efficiently across clusters, clouds, and data centers.
```
Comment (divya-mohan0209) [inline blog/what-is-a-Slice.mdx:14]:
```suggestion
It achieves this by creating a logical application boundary known as a **Slice** that allows pods and services to communicate efficiently across clusters, clouds, and data centers.  
```

File: blog/what-is-a-Slice.mdx (hunk 27 above)
Comment (divya-mohan0209) [reply blog/what-is-a-Slice.mdx:51]: @verma-kunal : This comment needs to be addressed, please.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Slight nits. When addressed, we can merge this & syndicate it for further distribution.

**Comment by verma-kunal** [comment]
Hey @divya-mohan0209 , I apologise for the delayed response. Was in a low network area for a few days, so wasn't able to work on the suggestions.
Will do the necessary & update here asap!

**Comment by verma-kunal** [comment]
> @verma-kunal : Gentle reminder here.

Hey @divya-mohan0209 , the required changes have been done!
Will make a PR shortly

**Comment by verma-kunal** [comment]
Hey @divya-mohan0209 , I'll actually be making a new PR for this as due to some reason, I am having troubles with merge conflict and commit-signing

**Comment by divya-mohan0209** [comment]
@verma-kunal That's okay. You can close this PR.
